modules/core/test/browser/browser.spec.ts

```diff
@@ -0,0 +1,222 @@
+        coin = bitgo.coin('teos');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('teos')
+    });
+    it("TETC should work", () => {
+        coin = bitgo.coin('tetc');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('tetc')
+    });
+    it("TETH should work", () => {
+        coin = bitgo.coin('teth');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('teth')
+    });
+    it("TLTC should work", () => {
+        coin = bitgo.coin('tltc');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('tltc')
+    });
+    it("TRBTC should work", () => {
+        coin = bitgo.coin('trbtc');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('trbtc')
+    });
+    it("TRMG should work", () => {
+        coin = bitgo.coin('trmg');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('trmg')
+    });
+    it("TRX should work", () => {
+        coin = bitgo.coin('trx');
+        expect(coin).toBeTruthy()
+        expect(coin.type).toEqual('trx');
+    });
+    it("TSUSD should work", () => {
+        coin = bitgo.coin('tsusd');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('tsusd')
+    });
+    it("TTRX should work", () => {
+        coin = bitgo.coin('ttrx');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('ttrx')
+    });
+    it("HBAR should work", () => {
+        coin = bitgo.coin('hbar');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('hbar')
+    });
+    it("THBAR should work", () => {
+        coin = bitgo.coin('thbar');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('thbar')
+    });
+    it("XTZ should work", () => {
+        coin = bitgo.coin('xtz');
+        expect(coin).toBeTruthy()
+        expect(coin.type).toEqual('xtz');
+    });
+    it("TXTZ should work", () => {
+        coin = bitgo.coin('txtz');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('txtz')
+    });
+    it("TXLM should work", () => {
+        coin = bitgo.coin('txlm');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('txlm')
+    });
+    it("TXRP should work", () => {
+        coin = bitgo.coin('txrp');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('txrp')
+    });
+    it("TZEC should work", () => {
+        coin = bitgo.coin('tzec');
+        expect(coin).toBeTruthy();
+        expect(coin.type).toEqual('tzec')
+    });
+    it("XLM should work", () => {
+        coin = bitgo.coin('xlm');
+        expect(coin).toBeTruthy()
+        expect(coin.type).toEqual('xlm');
+    });
+    it("XRP should work", () => {
+        coin = bitgo.coin('xrp');
+        expect(coin).toBeTruthy()
+        expect(coin.type).toEqual('xrp');
+    });
+    it("ZEC should work", () => {
+        coin = bitgo.coin('zec');
+        expect(coin).toBeTruthy()
+        expect(coin.type).toEqual('zec');
+    });
```

Comment:
can we actually add a function to draw from the statics coinmap? and then maybe add a decorator that's like "good for browsering" - this seems like it'll get forgotten about as we onboard new coins

Reply:
ACK